src/modules/blockchain/implementation/web3-service.js

```diff
@@ -237,82 +194,38 @@ class Web3Service {
         return this.config.blockTime;
     }
 
-    async deployIdentity() {
-        if (!this.config.identity) {
-            const transactionReceipt = await this.deployContract(Identity, [
-                this.getPublicKey(),
-                this.getManagementKey(),
-            ]);
-            this.config.identity = transactionReceipt.contractAddress;
-        } else {
-            this.logger.info(`Using existing identity: ${this.config.identity}`);
-        }
-    }
-
-    async profileExists(identity) {
-        const nodeId = await this.callContractFunction(this.ProfileStorageContract, 'getNodeId', [
-            identity,
-        ]);
-        return nodeId != null;
+    async profileExists() {
```

Comment:
Name of the function is profileExists and it's checking if identity exists, can you please rename it to identityExists